### DISCUSSION TITLE
There is a way for the program to classify articles from least relevant to most relevant?

### DISCUSSION THREAD

**Discussion by JhenniferNascimento** [asreview/asreview#1453] [Q&A]
I am using ASReview for article screening in my systematic literature review, however, during the review process, I have noticed that my search may contain more relevant articles than irrelevant ones, and I would like to know if there is a way for the program to classify articles from least relevant to most relevant.

If possible, I would like to explore this functionality as it would help me prioritize the review of the most relevant articles right from the start. Additionally, if there are any configurations or adjustments that I need to make to enable this classification, I would appreciate receiving instructions on how to proceed.

**Comment by Rensvandeschoot** [comment]
You can export the results of your labeling to a RIS, CSV, TSV, or Excel file. A file contains all imported data, including your decisions.
The following variables will be added to your dataset:

The column titled included contains the labels as provided by the user: 0 = not relevant, 1 = relevant and if missing it means the record is not seen during the screening process.

The column titled asreview_ranking contains an identifier to preserve the rank ordering as described below.

The file is ordered as follows:

- All relevant records you have seen in the order they were shown during the screening process.
- All records not seen during the screening and ordered from most to least relevant according to the last iteration of the model.
- All non-relevant records are presented in the order these are shown during the screening process.

Source [readthedocs](https://asreview.readthedocs.io/en/latest/progress.html#export-results)

**Comment by Rensvandeschoot** [comment]
For a detailed description of how ASReview stores data, see the pre-print paper: [Reproducibility and Data storage for Active Learning-Aided Systematic Reviews](https://psyarxiv.com/g93zf)
